README.md

```diff
@@ -1,18 +1,22 @@
 # Faraday
 
 Faraday is an HTTP client lib that provides a common interface over many
-adapters (such as Net::HTTP) and embraces the concept of Rack middleware when
+adapters (such as `Net::HTTP`, the default adapter) and embraces the concept of Rack middleware when
```

Comment:
This note wasn't necessary. Instead, the "default" indicator could be added to the list below.